app/views/user_mailer/invitation_to_claim.html.erb

```diff
@@ -3,13 +3,13 @@
     <%= t '.greeting', app_name: ArchiveConfig.APP_NAME %>
   </h2>
 
-  <p><%= t('.html.part1', open_doors_link: style_link(t('.html.open_doors'), "http://opendoors.transformativeworks.org"), app_short_name: ArchiveConfig.APP_SHORT_NAME, app_link: style_link(ArchiveConfig.APP_NAME, root_url)).html_safe %></p>
+  <p><%= t('.html.part1', open_doors: style_link(t('.html.open_doors_name'), "http://opendoors.transformativeworks.org"), app_short_name: ArchiveConfig.APP_SHORT_NAME, app_link: style_link(ArchiveConfig.APP_NAME, root_url)).html_safe %></p>
```

Comment:
Can we use the `_link` suffix throughout this file as well, i.e. 

* open_doors -> open_doors_name_link
* news_link -> ao3_news_link
* open_doors_faq_link -> faq_page_link
* open_doors_tutorial_link -> tutorial_page_link
* contact_support -> contact_support_text_link
* contact_open_doors -> contact_open_doors_text_link
* contact_support -> contact_support_text_link
* contact_support -> contact_support_text_link
* etc

(If the problem is the plain text version of this email is using `_link`, those should change to `_url`, since they're just written-out URLs, not hyperlinks.)

Reply:
Got it, and managed to do them all with no namespace clashes anyway